### ISSUE TITLE
Comments don't work when changing schema

### ISSUE THREAD

**Issue by pjebs** [dgraph-io/dgraph#3130]
```
	op.Schema = `
		user: bool .
		user.name: string @index(exact) . # this should be unique
		user.email: string @index(exact) . # this should be unique and lower-cased
		user.password: password .
		user.code: string . # for password recovery (can be null)

		node: bool .
		node.hashid: string @index(exact) . # @username/hashid
		node.owner: uid @reverse . # (can be null)
		node.parent: uid . # [uid] (use facet)
		node.xdata: string . # store custom json data
	`
```

rpc error: code = Unknown desc = Invalid ending

**Comment by pjebs** [comment]
When I remove the comments, it works

**Comment by MichelDiz** [comment]
Comments works only for Queries and Mutations.

**Comment by MichelDiz** [comment]
Closing due duplicate https://github.com/dgraph-io/dgraph/issues/1667

**Comment by manishrjain** [comment]
This should be an easy one for us to add.

**Comment by srfrog** [comment]
I was in the neighborhood (working with lexer) so I figure I'd get this one out. A lot of people, like me, prefer to document their schemas when building domain applications. So I think this is very useful.

**Comment by pjebs** [comment]
Will be released in v1.1?
